frontend/src/js/components/login/Password.tsx

```diff
@@ -129,7 +129,7 @@ export const Password = () => {
     });
 
   return (
```

Comment:
might be also nice to unwrap `dispatch(passwordResetStart(formData.email))`. Otherwise both failed and successfull requests will suggest user to wait for an email